unity-renderer/Assets/DCLServices/PlacesAPIService/NotAPlaceException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using UnityEngine;
+
+namespace DCLServices.PlacesAPIService
+{
+    public class NotAPlaceException : Exception
+    {
+        public NotAPlaceException(string placeUUID) : base($"Couldnt find place with ID {placeUUID}") { }
+        public NotAPlaceException(Vector2Int coords) : base($"Scene at {coords.ToString()} is not a place") { }
```

Comment:
`ToString` is redundant, it will converted to the string anyway in the same fasion